src/main/java/com/metamx/common/parsers/JSONPathParser.java

```diff
@@ -61,7 +63,10 @@ public JSONPathParser(List<FieldSpec> fieldSpecs, boolean useFieldDiscovery, Obj
     this.fieldPathMap = generateFieldPaths(fieldSpecs);
     this.useFieldDiscovery = useFieldDiscovery;
     this.mapper = mapper == null ? new ObjectMapper() : mapper;
-    this.jsonPathConfig = Configuration.defaultConfiguration().addOptions(Option.SUPPRESS_EXCEPTIONS);
+    this.jsonPathConfig = Configuration.builder()
```

Comment:
Add comment as to why defaultConfiguration pulls in json-smart and why that is bad